docs/fidesops/docs/guides/saas_postprocessors.md

```diff
@@ -0,0 +1,134 @@
+
+## Configuration
+
+Post-processors are configured within the `endpoints` section of a `saas_config`
+
+
+```yaml
+endpoints:
+  - name: messages
+    requests:
+      read:
+        path: /conversations/<id>/messages
+        request_params:
+          ...
+        postprocessors:
+          - strategy: unwrap
+            configuration:
+              data_path: conversation_messages
+          - strategy: filter
+            configuration:
+              field: from_email
+              value:
+                identity: email
+```
+
+Note: Order matters as its defined in the config. In the above example, unwrap will be run first, then the output of unwrap will be used in the filter strategy.
+
+
+## Supported Strategies
+- `unwrap`: Gets object at given data path.
+- `filter`: Filters data based on given field value.
+
+
+### Filter
+
+Filters object or array given field name and value. Value can reference a dynamic identity passed in through the request OR be a hard-coded value.
+
+#### Configuration Details
+
+`strategy`: filter
+
+`configuration`:
+
+- `field` (_str_): Corresponds to the field on which to filter. For example, we wish to filter where `email_contact == "bob@mail.com"`, then `field` will be `email_contact`. Note that filtering an array by a field that's deeply nested is not yet supported.
+- `value` (_str_): Value to search for when filtering (e.g. hard-coded `bob@mail.com`) or Dict of identity path:
+    - `identity` (_str_): Identity object from subject request (e.g. `email` or `phone_number`)
+
+
+#### Example
+
+Post-Processor Config:
+```yaml
+- strategy: filter
+  configuration:
+    field: email_contact
+    value:
+      identity: email
+```
+
+Identity data passed in through request:
+
+```
+{
+  "email": "somebody@email.com"
+}
+```
+
+Data to be processed:
+```
+data = [
+    {
+        "id": 1397429347
+        "email_contact": "somebody@email.com"
+        "name": "Somebody Awesome"
+    },
+    {
+        "id": 238475234
+        "email_contact": "somebody-else@email.com"
+        "name": "Somebody Cool"
+    }
+]
+```
+
+Note: Type casting is not supported at this time. We currently only support filtering by string values. e.g. `bob@mail.com` and not `12344245`.
+
+
+### Unwrap
+
+Given a path to a dict/list, returns the dict/list at that location.
+
+#### Configuration Details
+
+`strategy`: unwrap
+
+`configuration`:
+
+- `data_path` (_str_): Gives the path to desired object. E.g. `exact_matches.members` will attempt to get the `members` object on the `exact_matches` object.
+
+
+#### Example
+
+Post-Processor Config:
+```yaml
+- strategy: unwrap
+  configuration:
+    data_path: exact_matches.members
+```
+
+Data to be processed:
+```
+data = {
+  "exact_matches": {
+    "members": [
+      { "howdy": 123 },
+      { "meow": 841 }
+    ]
+  }
+}
+
+data_path = exact_matches.members
+    
+```
+Result:
+```
+result = [
+  { "howdy": 123 },
+  { "meow": 841 }
+]
+```
```

Comment:
I really like this `Result` section because it gives a clear input / output. Could we add something similar for `Filter` above too?

Reply:
forgot this section, thanks @seanpreston !